drivers/sensor/tdk/icp201xx/icp201xx_spi.c

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2025 TDK Invensense
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * Bus-specific functionality for ICP201XX accessed via SPI.
+ */
+
+#include "icp201xx_drv.h"
+
+#if CONFIG_SPI
```

Comment:
You already check this in your CMakeLists.txt, no need to add the condition here